products/java.md

```diff
@@ -125,6 +125,15 @@ releases:
     releaseDate: 2006-12-12
     link: https://www.oracle.com/java/technologies/javase/6-relnotes.html#R160_211
     latestReleaseDate: 2018-10-16
+    
+-   releaseCycle: "5"
+    lts: false
+    support: 2009-11-03
+    eol: 2009-11-03
+    latest: "1.5.0_85"
```

Comment:
```suggestion
    latest: "5.0u85"
```
In https://github.com/endoflife-date/endoflife.date/pull/2475#discussion_r1100126757 I thought changing like that.